plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -0,0 +1,1466 @@
+            type: str
+            default: "3"
+          timeout:
+            description:
+            - Number of seconds before timing out between devices and authentication and policy server.
+            - Timeout should be from 2 to 20.
+            type: str
+            default: "4"
+          role:
+            description:
+            - Role of authentication and policy server.
+            - Updation of role is not possible
+            type: str
+            default: secondary
+          pxgrid_enabled:
+            description:
+            - Set True to enable the Pxgrid and False to disable the Pxgrid.
+            - Pxgrid is available only for the Cisco ISE Servers.
+            - >
+              PxGrid facilitates seamless integration and information sharing across products,
+              enhancing threat detection and response capabilities within the network ecosystem.
+            type: bool
+            default: True
+          use_dnac_cert_for_pxgrid:
+            description: Set True to use the Cisco Catalyst Center certificate for the Pxgrid.
+            type: bool
+            default: False
+          cisco_ise_dtos:
+            description:
+            - List of Cisco ISE Data Transfer Objects (DTOs).
+            - Required when server_type is set to ISE.
+            type: list
+            elements: dict
+            suboptions:
+              user_name:
+                description:
+                - User name of the Cisco ISE server.
+                - Required for passing the cisco_ise_dtos.
+                type: str
+              password:
+                description:
+                - Password of the Cisco ISE server.
+                - Required for passing the cisco_ise_dtos.
+                type: str
+              fqdn:
+                description:
+                - Fully-qualified domain name of the Cisco ISE server.
+                - Required for passing the cisco_ise_dtos.
+                type: str
+              ip_address:
+                description:
+                - IP Address of the Cisco ISE Server.
+                - Required for passing the cisco_ise_dtos.
+                type: str
+              subscriber_name:
+                description:
+                - Subscriber name of the Cisco ISE server.
+                - Required for passing the cisco_ise_dtos.
+                type: str
+              description:
+                description: Description about the Cisco ISE server.
+                type: str
+              ssh_key:
+                description: SSH key of the Cisco ISE server.
+                type: str
+          external_cisco_ise_ip_addr_dtos:
+            description: External Cisco ISE Ip address data transfer objects for future use.
+            type: list
+            elements: dict
+            suboptions:
+              external_cisco_ise_ip_addresses:
+                description: External Cisco ISE Ip addresses.
```

Comment:
IP

Reply:
is it the virtual IP addresses? Why don't we move this inside to           cisco_ise_dtos: